pages/content/ch5/fields.mdx

```diff
@@ -335,3 +335,56 @@ Because we were able to eliminate the possibilities $1, 2, 3, 4, 6, 8, 12$ as or
 </div>
 </Problem>
 
+## Exercises
+
+<Problem title="MGO" source="Philipp Barski, Yannick Funke" difficulty={3} relevance={3}>
+    <div label="question">
+    What is the order of $\mathrm{GF}(p)[x]_{x^m}^*$? Prove your answer.
```

Comment:
... with $p$ being prime.

Reply:
... and $m \in \mathbb N$ oder so

Reply:
or p is prime power

Reply:
Dass folgt aus der Def von GF. Kann man aber noch erwähnen.

Reply:
nicht wirklich (gibt keine richtige Def im skript), und $m \in N$ folgt nicht daraus